test/e2e/utils.js

```diff
@@ -83,20 +38,14 @@ const currentContentType = async (page) => {
   return contentSwitcherButton.textContent()
 }
 
-const mockAllSearch = async (context) => {
-  await Promise.all([
-    mockAudioSearch(context),
-    mockImageSearch(context),
-    mockThumbnails(context),
-    mockImages(context),
-  ])
+const mockProviderApis = async (context) => {
+  await context.route('**.jamendo.com**', (route) => route.abort())
```

Comment:
I kept meaning to ask about this, but why do we only mock the jamendo provider? Wouldn't we want to mock all providers to stop e2e from actually hitting them if we're mocking any of them at all?

Reply:
I was mocking all the requests that end with `jpg`| `svg` | `png` before to make sure we stop hitting the providers. And it's not possible to mock the jamendo request by using an extension, because its URL doesn't have one. I think this should be another new issue: Configure mocking requests to providers for media.

Reply:
Opened the issue: #914